lib/helper/FileSystem.js

```diff
@@ -133,10 +169,47 @@ class FileSystem extends Helper {
   }
 }
 
-function getFileContents(file, encoding) {
+module.exports = FileSystem;
+
+/**
+ * @param {string} file
+ * @param {string} [encoding='utf8']
+ * @returns {string}
+ */
+function getFileContents(file, encoding = 'utf8') {
   if (!file) assert.fail('No files were opened, please use seeFile action');
-  encoding = encoding || 'utf8';
-  return fs.readFileSync(file, 'utf8');
+  if (encoding === '') assert.fail('Encoding is an empty string, please set a valid encoding');
+  return fs.readFileSync(file, encoding);
 }
 
-module.exports = FileSystem;
+/**
+ * @param {string} file
+ * @param {number} timeout
+ * @returns {Promise}
+ */
+function ifFileExists(file, timeout) {
```

Comment:
is
```suggestion
function isFileExists(file, timeout) {
```

Reply:
I thought it's a typo because of https://github.com/Codeception/CodeceptJS/pull/2164#discussion_r371089606

Reply:
oh, sorry